public/app/features/admin/state/apis.tsx

```diff
@@ -21,8 +26,25 @@ export interface ServerStat {
 }
 
 export const getServerStats = async (): Promise<ServerStat | null> => {
+  let resp = await getBackendSrv()
+    .get('/api/anonymous/stats')
+    .then((res) => {
+      return res;
+    })
+    .catch((err) => {
+      console.error(err);
+      // FIXME:
+      // return null;
+    });
   return getBackendSrv()
     .get('api/admin/stats')
+    .then((res) => {
+      if (resp) {
+        res.activeDevices = resp;
+        res.activeAnonymousUsers = resp / 3;
+      }
+      return res;
+    })
```

Comment:
We could issue both request and then use `Promise.all` to wait for both to complete and then merge the result. We don't have to wait for the first one before we issue next request.

I would also remove `activeAnonymousUsers` from here and then perform this where it is displayed. Then the check there will be simpler

Reply:
Also wrap these in a try catch block

Reply:
I did these in two requests, as i wanted redundancy as if one fails at least we display some information. In case the anon/stats fails for instance.

Reply:
To handle partail success https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/allSettled can be used